plugin/src/main/java/io/snyk/languageserver/protocolextension/InvalidProductTypeException.java

```diff
@@ -0,0 +1,7 @@
+package io.snyk.languageserver.protocolextension;
+
+public class InvalidProductTypeException extends Exception {
```

Comment:
nitpick: I'd extend RuntimeException to not having to pollute signatures and declare it everywhere.